### DISCUSSION TITLE
Request for access to Copilot Chat beta version

### DISCUSSION THREAD

**Discussion by wenxiayili** [community/community#55625] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hello,
I am writing to express my deep interest in Copilot Chat, a new feature of Github that uses AI to help users write code faster. I have been following the development of this exciting new feature closely and I believe it will significantly improve my coding experience.

I am writing to kindly ask if it is possible for me to gain access to the beta version of Copilot Chat. I am very eager to try out this new feature and offer my feedback on how it can be further improved.

<Edited by Staff> Please let me know if there is any additional information you need from me to grant me early access to Copilot Chat.

Thank you for considering my request. I look forward to hearing from you soon.

**Comment by Akash1134** [comment]
Hi @wenxiayili  ,

Thank you for showing interest in Copilot Chat. While we are unable to guarantee access to feature(s) in the previews, you can [submit your interest by joining the waitlist](https://github.com/features/preview/copilot-x) to preview the features that you are interested in. Joining one waitlist does not confirm your interest in more than one feature, so you must join each separately. Please note we currently don’t have guaranteed timelines for access. 

Once you get access to preview a feature, please keep in mind that what you are getting access to is considered beta or a technical preview, meaning the feature may still have some kinks to work out.

**Comment by jianlinshi** [comment]
It seems the url redirected to purchase page.  Can I have access to the Enterprise copilot if we have one? Thanks.
